RELEASE_PROCEDURE.md

```diff
@@ -17,10 +18,19 @@
      - NEVER do:
        - Add new features
 
+   - Once the release branch is ready to be merged, create a new Pull Request from the release branch into `master`. This should only be used to exercise CI and for the rest of the team the approve that all necessary changes have been made for release, and **not for doing the actual merge**. **The actual merge into master should be done locally in the next step.**
```

Comment:
I think some extra description of the new 2 PR release process would be helpful here:

```suggestion
  - Now commit and push the changes to `release-X.X.X-changes`

```shell
 git commit release-X.X.X-changes -m "bumped version"
 git push -u origin release-X.X.X-changes```

- Make a PR from `release-X.X.X-changes` into `release-X.X.X` and merge once approved

   - Once the `release-X.X.X` branch is ready to be merged, create a new Pull Request from the release branch into `master`. This should only be used to exercise CI and for the rest of the team the approve that all necessary changes have been made for release, and **not for doing the actual merge**. **The actual merge into master should be done locally in the next step.**
```